### DISCUSSION TITLE
Sign out callback

### DISCUSSION THREAD

**Discussion by Rees1993** [nextauthjs/next-auth#3673] [Help] [answered]
I have a potential scenario where I need to clear data from a store once a user is logged out? 

Could having a sign out callback so I can add custom logic in there help with this or is there another way?

**Comment by balazsorban44** [comment, marked as answer]
https://next-auth.js.org/configuration/events#signout

**Reply by Rees1993** [reply]
Of course, Events! I just thought of them as audit logs as mentioned in the docs but this makes more sense. Thanks!

**Reply by balazsorban44** [reply]
I understand the confusion. If you want to clarify it, please open a PR! (Edit button at the bottom of the docs page)

**Reply by AhmedBaset** [reply]
Will the `signOut` event be triggered before or after signing out?

**Reply by Ali-hd** [reply]
before
